lib/manager/nuget/index.ts

```diff
@@ -9,7 +9,7 @@ export const defaultConfig = {
   fileMatch: [
     '\\.(?:cs|fs|vb)proj$',
     '\\.(?:props|targets)$',
-    '\\.config\\/dotnet-tools\\.json$',
+    '(^|\\/)dotnet-tools\\.json$',
     '(^|//)global\\.json$',
```

Comment:
```suggestion
    '(^|\/)global\\.json$',
```

We should fix this too, as it's wrong

Reply:
Done